src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Ball {
+    private String colour;
+    private int number;
+
+    public Ball(Color randomColor, int randomNumber) {
+    }
```

Comment:
Why do we have a constructor with unused arguments?
Please assign incoming values to object